features/dapp_develop/dapp_develop.feature

```diff
@@ -44,5 +44,5 @@ Scenario Outline: DApp developer uses config states, compiles, deploys and invok
 
   Examples: 
         | Tool         | ContractExampleSubPath | ContractName                  | ContractCompiledFileName      | FunctionName     | FunctionParams                              | RootIdentityName  | TesterIdentityName  | NetworkConfigName   | Result |
-#       | JSSDK        | auth                   | soroban-auth-contract         | soroban_auth_contract.wasm    | increment        | --user <tester_identity_pub_key> --value 2  | r1                | t1                  | standalone          | 2      |
+#       | NODEJS       | auth                   | soroban-auth-contract         | soroban_auth_contract.wasm    | increment        | --user <tester_identity_pub_key> --value 2  | r1                | t1                  | standalone          | 2      |
```

Comment:
just wondering, what if copy this line up to the `DApp developer compiles, deploys and invokes a contract` scenario..that one doesn't use config states during invocations, which is the part that doesn't translate into js yet correct?

Reply:
I think this is using a separate pubkey than we are above, though, so maybe the auth wouldn't work?

Reply:
yes, to have the scenario pass the root pubkey for the tx and the `--user` fn param, looks a mod to `invokeInvokerAuthContractStep` to support a new `root_identity_pub_key` substitution tag that resolves to  `testConfig.E2EConfig.TargetNetworkPublicKey` similar to how `tester_identity_pub_key` resolves to the config identity state via `testConfig.Identities[identity]`

can add that coverage as later iteration, not to hold this up.